packages/create-project/commands/init/index.js

```diff
@@ -68,18 +71,31 @@ module.exports = class Project extends Generator {
 
   _initialPackageJson() {
     const installer = isYarn ? 'yarn' : 'npm';
-    const scripts = { build: `${packages.NEUTRINO} build` };
+    const scripts = { build: 'webpack --mode production' };
 
     if (this.data.projectType !== 'library') {
-      scripts.start = `${packages.NEUTRINO} start`;
+      scripts.start = this.data.project === '@neutrinojs/node'
+        ? 'webpack --watch --mode development'
+        : 'webpack-serve --mode development';
```

Comment:
Running `yarn start` on a create-project with options React+Jest+AirBnb, I get:
```
$ yarn start
yarn run v1.6.0
(node:6040) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
$ webpack-serve --mode development
× ｢config｣: An error ocurred while trying to find a config file
              Did you forget to specify a --require?
C:\Users\Ed\src\test-project\node_modules\@webpack-contrib\config-loader\lib\load.js:84
    throw new LoadConfigError(e, configPath);
    ^

LoadConfigError: Cannot find module 'webpack-dev-server/client'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:571:15)
    at Function.require.resolve (C:\Users\Ed\src\test-project\node_modules\webpack-serve\node_modules\v8-compile-cache\v8-compile-cache.js:162:23)
    at config.entry.batch.entry (C:\Users\Ed\src\test-project\node_modules\@neutrinojs\web\index.js:235:43)
```

I guess we have no coverage of this at the moment? 

Perhaps we can do either (or both) of:
* test webpack build using `--mode development`
* see if we can spawn the dev server via `yarn start`, then if it starts successfully, kill it after a few seconds

Until then perhaps we need to hand-test `(using create-project and `--registry`) all of the top level preset's `yarn start` commands?

Also, I think the reason the unit tests didn't catch this is because the repo root `package.json` contains `webpack-dev-server` (when I think it perhaps shouldn't?).

Reply:
Oooooooh, this is a problem. I don't think we can use webpack-serve out-of-the-box because of our hardcoded entries in `devServer`:

https://github.com/mozilla-neutrino/neutrino-dev/blob/master/packages/web/index.js#L235

Reply:
Perhaps we should switch back to webpack-dev-server for now (also reduces the amount changed in one PR) and we can figure it out later?

Reply:
You read my mind.